packages/catalog-generator/src/main/java/io/kaoto/camelcatalog/maven/KaotoMavenVersionManager.java

```diff
@@ -126,7 +124,10 @@ public void resolve(MavenDownloader mavenDownloader, String gav, boolean useSnap
                 ((KaotoOpenURLClassLoader) getClassLoader()).addURL(ma.getFile().toURI().toURL());
             }
         } catch (Throwable e) {
-            e.printStackTrace();
+            if (getLog()) {
```

Comment:
what' the purpose of getLog()?

Reply:
Dön't ask me :D I just aligned to the rest of the codebase. Will leave that to Ricardo.

Reply:
Is to use the same log property in both classes (super and current one)